### ISSUE TITLE
how to set text size?

### ISSUE THREAD

**Issue by ggggxiaolong** [GoodieBag/Pinview#19]
how to set text size?

**Comment by FarshidRoohi** [comment]
you can use attribute textSize in xml file

```
<com.goodiebag.pinview.Pinview
        android:id="@+id/pinview"
        app:pinBackground="@drawable/example_drawable"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        app:pinWidth="40dp"
        app:pinHeight="40dp"
        app:pinLength="4"
        app:cursorVisible="false"
	app:forceKeyboard="true"
        app:hint="0"
        app:inputType="text"
        app:password="false"
        app:textSize="12sp"
/>
```

**Comment by Vatican-Cameos** [comment]
Available in v1.4
